tools/mo/openvino/tools/mo/front/common/partial_infer/eltwise.py

```diff
@@ -80,6 +81,34 @@ def broadcast_dims(dim1, dim2):
             node.out_port(0).data.set_value(op(node.out_node().value, values[i + 1]))
 
 
+def eltwise_reverse_infer(node: Node):
+    input_1_shape = node.in_port(0).data.get_shape()
+    input_2_shape = node.in_port(1).data.get_shape()
+    output_shape = node.out_port(0).data.get_shape()
+
+    if output_shape is not None:
+        out_rank = len(output_shape)
+        deduced_in_shape = undefined_shape_of_rank(out_rank)
+
+        if input_1_shape is not None and input_2_shape is None and out_rank > len(input_1_shape):
+            in_port_to_update = 1
+            defined_in_shape = input_1_shape
+        elif input_2_shape is not None and input_1_shape is None and out_rank > len(input_2_shape):
+            in_port_to_update = 0
+            defined_in_shape = input_2_shape
+        else:
+            return None
+        defined_in_rank = len(defined_in_shape)
+
+        # in defined_input_shape = [1] and output_shape = [x, 400, 400, 3]
+        # we should not loose these information
```

Comment:
typo: 'in' -> 'if'
rephrase: 'we should not loose these information' -> 'partial shape information about sizes should be lost'

Reply:
fixed